### PR TITLE
Fix example code in readme.

### PR DESCRIPTION
**Pull request by rettichschnidi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +25,11 @@ int main()
...
     TabSample tab;
     // explicit all_of(tab)
-    for(const auto& row : db.run(select(all_of(tab)).from(tab).where(true)))
+    for(const auto& row : db(select(all_of(tab)).from(tab).where(true)))
     {
         int64_t alpha = row.alpha;
-        bool beta = row.beta;
-        std::string gamma = row.gamma;
+        std::string beta = row.beta;
+        bool gamma = row.gamma;
```
Comment (rbock) [inline README.md]: Why did you change the types here?

Comment (rettichschnidi) [inline README.md]: To match the definitions in [TabSample.h](https://github.com/rbock/sqlpp11-connector-sqlite3/blob/master/tests/TabSample.h#L107)

Guess I should update the table definition at line 20/21 as well?

Comment (rbock) [inline README.md]: I am a bit confused as to why the test even works, but that might be due to the specific values in insert? Weird.

But yes, lines 20/21 should be changed as well, as should the definition in SampleTest.cpp

Comment (rettichschnidi) [inline README.md]: Gonna do that.

Question: Would you rather have patches for the branch develop instead of master?

Comment (rbock) [inline README.md]: patches to develop are preferred in most cases. Since this is primarily a bug fix for the readme, master is just fine.

Thanks!

Comment (rettichschnidi) [inline README.md]: TIL: [SQLite lets me insert a string into a database column of type integer!](https://www.sqlite.org/faq.html#q3)

**Comment by rbock** [comment]
Thanks for the fix and the explanation :-)
